integrations/kube_apis/kube_apis.go

```diff
@@ -26,6 +29,15 @@ const (
 	POSTMAN_INSIGHTS_K8S_NODE = "POSTMAN_INSIGHTS_K8S_NODE"
 )
 
+// An exponential retry backoff policy that results in approximately 24 retires over a 24 hour period.
+var retry = wait.Backoff{
+	Duration: 10 * time.Millisecond,
+	Factor:   2.0,
+	Jitter:   0.1,
+	Steps:    24,
+	Cap:      24 * time.Hour,
```

Comment:
I think this is probably unnecessarily large?

Retrying once per hour is probably slow enough for the worst-case backoff.